books/SPARK_for_the_MISRA_C_Developer/chapters/04_strong_typing.rst

```diff
@@ -1,4 +1,4 @@
-:code-config:`accumulate_code=False;run_button=True`
+:code-config:`run_button=True;prove_button=False;accumulate_code=False`
```

Comment:
same as above, if you do this, you'll end up with "Run" buttons for snippets that have no main

Reply:
yes, which corresponds to a compilation of those snippets, excactly what I want